drivers/serial/uart_realtek_rts5912.c

```diff
@@ -31,6 +34,76 @@ struct uart_rts5912_device_config {
 struct uart_rts5912_dev_data {
 };
 
+DT_INST_FOREACH_STATUS_OKAY(PINCTRL_DT_INST_DEFINE);
+#if defined(CONFIG_PM)
+#include <reg/reg_gpio.h>
+#include "zephyr/drivers/gpio/gpio_rts5912.h"
```

Comment:
can this be included all the time? we normally avoid mid-file includes unless it's like a blob struct or something like that